runtime/sema/type_tags.go

```diff
@@ -219,6 +219,9 @@ const (
 	anyResourceAttachmentMask
 	anyStructAttachmentMask
 
+	// This is here because we didn't reserve type-tags for Word128 and Word256.
+	word128TypeMask
```

Comment:
@SupunS Type tags and masks are not stored right? Could we just move  this up?

Reply:
FYI - The lower mask has reached its limit. So to move this up, we'll have to move something down into the upper mask section.

Reply:
Correct!

In most places we are not able to re-arrange things in such a way, because if there is existing data stored in the execution state, such a change would change the meaning of what is stored. 

If types/tags are not stored, but only exist at checking time in memory, then we could easily re-arrange entries in the lower and upper masks how we see fit.

Reply:
Yes, this is safe to re-arrange. Type tags are not stored.

The reserved positions are just to make it less cumbersome when adding new types. Please feel free to update this.

Reply:
done, moved it to lower mask.